app/models/user.rb

```diff
@@ -22,21 +22,32 @@
 #
 class User < ApplicationRecord
 
+  # TODO: Move this to Global Application Config class
+  THEMES ||= %w[system dark light].freeze
```

Comment:
Why are we using pipes here? Constants get loaded when we boot up the application